AuthenticationApp/Controller/LoginVC.swift

```diff
@@ -22,17 +22,17 @@ class SignInVC: UIViewController {
     }
     
     @IBAction func signInButtonTapped(_ sender: Any) {
-        guard isValidUserData() else { return }
+//        guard isValidUserData() else { return }
         goToProfile()
     }
 }
 
-private extension SignInVC {
+private extension LoginVC {
     func setupUI() {
-        title = "Sign In"
+        title = "Login"
         setupLabels()
         setupTextFields()
-        signInButtonTapped.setTitle("Sign In", for: .normal)
+        signInButtonTapped.setTitle("Login", for: .normal)
```

Comment:
Please rename the outlets and actions to `login`.
```suggestion
        loginButtonTapped.setTitle("Login", for: .normal)
```

Reply:
Done.